bindgen/templates/EnumTemplate.cs

```diff
@@ -6,7 +6,6 @@
 // C# doesn't support enums with associated data. Use regular `enum` for flat
 // enums, and `record` for enums with associated data.
 #}
-{%- let e = ci.get_enum_definition(name).unwrap() %}
```

Comment:
I’m assuming this is supposed to be defined in the including template?

Reply:
It is, I just copy pasted from upstream :shrug: